resources/vue/src/router.js

```diff
@@ -0,0 +1,37 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import Account from "./components/account/common/Account";
+import MePage from "./components/account/me/MePage";
+import OrdersPage from "./components/account/orders/OrdersPage";
+import MessagesPage from "./components/account/messages/MessagesPage";
+import Site from "./components/site/common/Site";
+import MainPage from "./components/site/main/MainPage";
+import TripsPage from "./components/site/trips/TripsPage";
+import HotelsPage from "./components/site/hotels/HotelsPage";
+import ContactsPage from "./components/site/contacts/ContactsPage";
+import AboutPage from "./components/site/about/AboutPage";
```

Comment:
Move all of the components you are using here to `pages/` folder on the level of `components/`